src/vulkan/rt64_vulkan.h

```diff
@@ -16,7 +16,11 @@
 #elif defined(__ANDROID__)
 #define VK_USE_PLATFORM_ANDROID_KHR
 #elif defined(__linux__)
+#if defined(RT64_SDL_WINDOW_VULKAN)
+#define VK_USE_PLATFORM_WAYLAND_KHR
+#else
 #define VK_USE_PLATFORM_XLIB_KHR
+#endif
```

Comment:
Same as above, only wrap the XLIB one under `#if !defined(RT64_SDL_WINDOW_VULKAN)` and take out the Wayland one.

Reply:
This is now done as you told me, too.